examples/time/virtual_time.rs

```diff
@@ -0,0 +1,219 @@
+                top: Val::Px(0.),
+                padding: UiRect::all(Val::Px(20.0)),
+                ..default()
+            },
+            ..default()
+        })
+        .with_children(|builder| {
+            // real time info
+            builder.spawn((
+                TextBundle::from_section(
+                    "",
+                    TextStyle {
+                        font_size,
+                        ..default()
+                    },
+                ),
+                RealTime,
+            ));
+
+            // keybindings
+            builder.spawn(
+                TextBundle::from_section(
+                    "CONTROLS\nUn/Pause: Space\nSpeed+: +/Up\nSpeed-: -/Down",
+                    TextStyle {
+                        font_size,
+                        color: Color::rgb(0.85, 0.85, 0.85),
+                        ..default()
+                    },
+                )
+                .with_text_alignment(TextAlignment::Center),
+            );
+
+            // virtual time info
+            builder.spawn((
+                TextBundle::from_section(
+                    "",
+                    TextStyle {
+                        font_size,
+                        color: virtual_color,
+                        ..default()
+                    },
+                )
+                .with_text_alignment(TextAlignment::Right),
+                VirtualTime,
+            ));
+        });
+}
+
+/// Move sprites using `Real` (unscaled) time
+fn move_real_time_sprites(
+    mut sprite_query: Query<&mut Transform, (With<Sprite>, With<RealTime>)>,
+    // `Real` time which is not scaled or paused
+    time: Res<Time<Real>>,
+) {
+    for mut transform in sprite_query.iter_mut() {
+        // move roughly half the screen in a `Real` second
+        // when the time is scaled the speed is going to change
+        // and the sprite will stay still the the time is paused
+        transform.translation.x = get_sprite_translation_x(time.elapsed_seconds());
+    }
+}
+
+/// Move sprites using `Virtual` (scaled) time
+fn move_virtual_time_sprites(
+    mut sprite_query: Query<&mut Transform, (With<Sprite>, With<VirtualTime>)>,
+    // the default `Time` is either `Time<Virtual>` in regular systems
+    // or `Time<Fixed>` in fixed timestep systems so `Time::delta()`,
+    // `Time::elapsed()` will return the appropriate values either way
+    time: Res<Time>,
+) {
+    for mut transform in sprite_query.iter_mut() {
+        // move roughly half the screen in a `Virtual` second
+        // when time is scaled using `Time<Virtual>::set_relative_speed` it's going
+        // to move at diffent pace and the sprite will stay still when time is
+        // `Time<Virtual>::is_paused()`
+        transform.translation.x = get_sprite_translation_x(time.elapsed_seconds());
+    }
+}
+
+fn get_sprite_translation_x(elapsed: f32) -> f32 {
+    // move 500px in a second (either real or virtual seconds based on where elapsed comes from)
```

Comment:
I don't think this comment is true. sin(1.0) is ~420

Reply:
Oh, right.
Should I make it
_roughly 420px_
or 
_500px in roughly 1.5 seconds_
or just drop the comment?

Reply:
I prefer the second, or not talking about exactly how far/fast it will be moving.

Reply:
To be fair, this wasn't planned unless my masterplan was waiting for someone to point out my math blunder to reveal the this masterpiece of comedy.
Anyway, you are probably right, I'm just gonna drop the comment.

BTW. Isn't rust the ideal lang for this as of release 69?